### PR TITLE
[WIP] Russian localization

### PR DESCRIPTION
**Pull request by acedened**
- [x] Main.strings
- [ ] Localizable.strings

I'm not sure about some terms, would need to check it in simulator later

### CODE REVIEW

**Comment by acedened** [comment]
No problem! Could you please provide sources of the app with ability to test these localizations? I can guess where most of the strings are used, but others would be localized a way better if I could check them right in the app

**Comment by acedened** [comment]
Oh, okay then!

File: en-GB.lproj/Main.strings [hunk 1]
```diff
@@ -84,19 +84,19 @@
...
-"GIt-S3-wBe.text" = "Reload Daemons";
+"GIt-S3-wBe.text" = "Перезагрузить демоны";
 
 /* Class = "UILabel"; text = "Read-Only RootFS"; ObjectID = "GsM-At-dw9"; */
-"GsM-At-dw9.text" = "Read-Only RootFS";
+"GsM-At-dw9.text" = "RootFS только для чтения";
 
 /* Class = "UIButton"; normalTitle = "Restart"; ObjectID = "HS8-A8-Kos"; */
-"HS8-A8-Kos.normalTitle" = "Restart";
+"HS8-A8-Kos.normalTitle" = "Перезагрузка";
```
Comment (valery-kirichenko) [inline en-GB.lproj/Main.strings]: I think it should be the verb "Перезагрузить" as it's button's title

File: en-GB.lproj/Main.strings [hunk 2]
```diff
@@ -105,85 +105,85 @@
...
 /* Class = "UILabel"; text = "Refresh Icon Cache"; ObjectID = "SVp-PX-97M"; */
-"SVp-PX-97M.text" = "Refresh Icon Cache";
+"SVp-PX-97M.text" = "Обновить кэш иконок";
 
 /* Class = "UITextField"; placeholder = "XXXXXXXXXXXXXXXX"; ObjectID = "T5g-8V-gBQ"; */
 "T5g-8V-gBQ.placeholder" = "XXXXXXXXXXXXXXXX";
 
 /* Class = "UILabel"; text = "Uptime"; ObjectID = "TmA-aA-qbT"; */
-"TmA-aA-qbT.text" = "Uptime";
+"TmA-aA-qbT.text" = "Аптайм";
```
Comment (valery-kirichenko) [inline en-GB.lproj/Main.strings]: You can use "Время работы" to avoid usage of anglicism

File: en-GB.lproj/Main.strings (hunk 1 above)
Comment (acedened) [reply en-GB.lproj/Main.strings]: Thanks, fixed

File: en-GB.lproj/Main.strings (hunk 2 above)
Comment (acedened) [reply en-GB.lproj/Main.strings]: Fixed

**Comment by acedened** [comment]
Looks like this PR is no longer needed after #25

**Pull request closed without merging**